src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+        func|anat|dwi/
+            <source_keywords>[_referencemap-<referencemap>space-<space>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-ACPC_dseg.nii.gz
+            sub-001_space-ACPC_dseg.json
+```
+
+A segmentation could be a binary mask that functions as a discrete `label` for a
+single structure. In this case, the label key must be used to specify the
+corresponding structure. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-ACPC_label-GM_dseg.nii.gz
+```
+
+See "Anatomical labels" for reserved key values for `label`.
+
+### Probabilistic Segmentations
+
+Probabilistic segmentations of brain tissue represent a single tissue class with
+values ranging from 0 to 1 in individual 3D volumes or across multiple frames.
+Similarly to a discrete, binary segmentation, the label key can be used to
+specify the corresponding structure.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_keywords>[_referencemap-<referencemap>space-<space>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-ACPC_label-BG_probseg.nii.gz
+            sub-001_space-ACPC_label-WM_probseg.nii.gz
+```
+
+See "Anatomical labels" for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-ACPC_probseg.nii.gz
+		    sub-001_space-ACPC_probseg.json
+```
+
+The JSON sidecar must include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` need to map to Abbreviations defined in "Anatomical Labels".
+
+### Surface Parcellations
+
+Discrete parcellations (surface segmentations) of cortical structures should be
+stored as GIFTI or CIFTI file.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat/
+            <source_keywords>[_hemi-{L|R}][_referencemap-<referencemap>]_dseg.{label.gii|dlabel.nii}
+```
+
+The REQUIRED extension for GIFTI parcellations is `.label.gii`. The hemi tag is
+REQUIRED for GIFTI files. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_hemi-L_dparc.label.gii
+            sub-001_hemi-R_dparc.label.gii
+```
+
+The REQUIRED extension for CIFTI parcellations is `.dlabel.nii`. For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_dparc.dlabel.nii
+            sub-001_dparc.dlabel.nii
+```
+
+### Anatomical Labels
+
+BIDS supplies a standard, generic label-index dictionary, defined in the table
+below, that contains common tissue classes and can be used to map segmentations
```

Comment:
it would be good if this mapping were not only defined here, but would refer to an external (published) definition, e.g. as already in use in AFNI, FreeSurfer, or some other software tool.

Reply:
If you can point out to such external definitions I would be happy to add them.

Reply:
This is not really my expertise, but I am thinking about AFNI bricks, AAL labels and/or FS labels. E.g. the first two columns of https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/AnatomicalROI/FreeSurferColorLUT.

Over the years, I have gathered some of these mappings (between integers and anatomical labels) in the https://github.com/fieldtrip/fieldtrip/blob/master/fileio/ft_read_atlas.m function; please note that that function is not authoritative.

I think that RadLex could serve as authoritative listing of such a mapping, see e.g. http://www.radlex.org/RID/RID13495. 

These are listed in the http://www.radlex.org/RID/RID13495